sat/cli/jobstat/main.py

```diff
@@ -44,11 +45,16 @@ def wrap_long_row_lines(row):
         None. Modifies row in place.
     """
     if row.get('apid'):
-        row['apid'] = textwrap.fill(text=row['apid'], width=20)
+        row['apid'] = row['apid'][:8]+"..."
```

Comment:
nit: try to use single quotes more consistently. I also would prefer spaces around the `+` operator as a small style suggestion.
```suggestion
        row['apid'] = row['apid'][:8] + '...'
```